.github/workflows/ci.yml

```diff
@@ -108,6 +108,21 @@ jobs:
       with:
         command:  check
         args: --all --features tokio02
+ 
+  check_io_safety_feature:
+    name: Check io_safety feature
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Doesn’t support MacOS?

Reply:
The changes I wrote only impact `cfg(unix)` and `cfg(windows)`, so I included a Unix platform (`ubuntu-latest`) and a Windows platform (`windows-latest`). If you would like me to also test on MacOS I can.